### DISCUSSION TITLE
The issue with source maps- "Source code was not found"

### DISCUSSION THREAD

**Discussion by LushGojani** [getsentry/support#97] [Support]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Throw one error
2. That error is captured by sentry SDK
3. Look at specific issues at sentry.io
4. It shows this error message "Source code was not found (see Troubleshooting for JavaScript)".

### Expected Result

I want to get an unminified stack trace like in this image.
![unminified](https://user-images.githubusercontent.com/84705173/191857098-37776c6d-b61a-4c32-ba71-0da44e454bac.png)

### Actual Result

This is how it is showing stack trace now 
![SouceMaps](https://user-images.githubusercontent.com/84705173/191856606-cc5036ab-5069-4b75-bca0-18eb7aff4d5b.png)

It is showing this error message "Source code was not found (see Troubleshooting for JavaScript)".
I am working on the Nextjs project, using Next Js SDK of sentry.
 
Can someone help me to get the unminified stack trace?

**Comment by souredoutlook** [comment]
@LushGojani can you try using the sentry-cli command `sentry-cli sourcemaps explain <event_id>` as described [here](https://docs.sentry.io/platforms/javascript/sourcemaps/troubleshooting_js/) and let me know what it says.

**Reply by LushGojani** [reply]
After running the command sentry-CLI sourcemaps explain <event_id>, it shows this error message "Release has no artifacts uploaded".

![artifacts11](https://user-images.githubusercontent.com/84705173/192378675-c89f4716-bcaa-4eaf-829f-05d4c40e84ef.png)

**Reply by souredoutlook** [reply]
Hey there - it looks like we were unable to fetch the source context for that event because it's coming from a localhost address. You can circumvent this error by uploading source-maps when running a build locally.

If you have uploaded source maps I'll need a link to the event in Sentry so we can investigate further
